src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -106,9 +106,10 @@ private void processZipFile(CloudBlobContainer container, String zipFilename)
                 }
             }
 
-            List<ScannableItem> scannedItems = envelopeProcessor.processEnvelope(metadataStream);
+            Envelope envelope = envelopeProcessor.processEnvelope(metadataStream);
```

Comment:
Since you are now returning the envelope and not retrieving scannable I think you don't need transactional annotation here ?